Core.Tests/StateManagerTests.cs

```diff
@@ -0,0 +1,21 @@
+using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mark this assembly with 'System.Runtime.InteropServices.ComVisibleAttribute'](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931948)